src/OpenApi/Model/Components.php

```diff
@@ -21,8 +23,10 @@ final class Components
     private $responses;
     private $parameters;
     private $examples;
+    /** @SerializedName("requestBodies") */
```

Comment:
You shouldn't rely on annotations, this creates a hard dependency to `doctrine/annotations` (this will not work if this lib isn't installed). You should probably use the XML format instead.